tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -415,3 +422,1405 @@ def test_date_range(self):
             )
 
             assert len(response.data) == 0
+
+
+class GroupUpdateTest(APITestCase, SnubaTestCase):
```

Comment:
(Non blocking): One option here could be to just move the tests to work on `update_groups` directly, and do a couple of basic integration tests on each of the api endpoints. Saves us on duplication

Reply:
yeah, i agree. will probably add a ticket to come back to this so that I can unblock frontend work

Reply:
added: https://getsentry.atlassian.net/browse/APP-1054